news/7680.removal

```diff
@@ -0,0 +1 @@
+removed unused module test_all_pip and its associated folder.
```

Comment:
```suggestion
Remove unused ``tests/scripts/test_all_pip.py`` test script and the ``tests/scripts`` folder.
```

Reply:
Thanks a lot for all the help @chrahunt , I hope the commit is as per the requirement now , feel free to suggest if any more changes needed.